utilities/docker/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM openvswitch/ovn:2.12_e60f2f2_debian_master
```

Comment:
2.12 is rather old version, before OVS and OVN split.

Reply:
I used that image as the starting point as it was available in dockerhub at https://hub.docker.com/search?q=openvswitch%2Fovn&type=image, I am hoping that OVN images get regulary pushed as per the request at https://github.com/ovn-org/ovn/issues/67